### DISCUSSION TITLE
Map gets stuck on executionPlan Compile

### DISCUSSION THREAD

**Discussion by Krnlsoft** [LuckyPennySoftware/AutoMapper#3760] [NA]
Hi,
for a complex map with many navigation properties and parent child references,
the compilation of the execution plan gets stuck.

I'm using .net 6 with EF 6, not sure how do I debug the issue.
Obviously putting Ignore on all navigation property solve the issue.
ExplicitExpansion doesn't help.

https://github.com/AutoMapper/AutoMapper/blob/bdc0120497d192a2741183415543f6119f50a982/src/AutoMapper/Configuration/MapperConfiguration.cs#L168

It's hard for me to provide a PoC to use for testing since the project is very big.

**Comment by lbargaoanu** [comment]
https://github.com/AutoMapper/AutoMapper/discussions/3753#discussioncomment-1471375
